db/migrate/20220727132450_validate_not_null_champs_type_de_champ_id.rb

```diff
@@ -0,0 +1,7 @@
+class ValidateNotNullChampsTypeDeChampId < ActiveRecord::Migration[6.1]
+  def change
+    validate_check_constraint :champs, name: "champs_type_de_champ_id_null"
+    change_column_null :champs, :type_de_champ_id, false
+    remove_check_constraint :champs, name: "champs_type_de_champ_id_null"
```

Comment:
On ne conserve pas la constraint car doublon avec la FK ?
Et il ne vaudrait pas mieux prévoir l'expression `"type_de_champ_id IS NOT NULL"` pour rollback au cas ?
```suggestion
    remove_check_constraint :champs, "type_de_champ_id IS NOT NULL", name: "champs_type_de_champ_id_null"
```

Reply:
Tout ce code, je l'ai littéralement copié-collé depuis le message d'erreur de la gem `strong_migrations`. Si tu préfères vraiment ta solution, je peux changer.